smart_campus/app/urls.py

```diff
@@ -26,5 +26,10 @@
     url('^get_unanswered_question/$', views.get_unanswered_question,
         name='Get Unanswered questsion'),
     url('^add_answered_question/$', views.add_answered_question,
-        name='Add Answered Question')
+        name='Add Answered Question'),
+    url('^activate/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
+        views.activate, name='activate'),
+    url('^reset_password/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
+        views.reset_password_page, name='reset password page'),
+    url('^reset_password/$', views.reset_password, name="reset password")
```

Comment:
Follow the style of views above.
This name should be 'Reset Password'.
Line 31 and 33 should be modified as well.

Reply:
Okay, fixed.